src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/ApplicationInsights/RemoveApplicationInsights.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ApplicationInsights.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ApplicationInsights
+{
+    [Cmdlet(VerbsCommon.Remove, ApplicationInsightsNounStr, SupportsShouldProcess = true)]
```

Comment:
@frankguodongchen all cmdlets should have an output type, even if it is `void`

Reply:
updated the output type as bool, thanks!